### PR TITLE
Move `TablesMigrate` to a separate module

### PR DESCRIPTION
**Pull request by FastLee**

### CODE REVIEW

File: src/databricks/labs/ucx/cli.py [hunk 1]
```diff
@@ -11,9 +11,9 @@
 from databricks.labs.ucx.framework.tui import Prompts
 from databricks.labs.ucx.hive_metastore import ExternalLocations, TablesCrawler
 from databricks.labs.ucx.hive_metastore.mapping import TableMapping
-from databricks.labs.ucx.hive_metastore.tables import TablesMigrate
 from databricks.labs.ucx.install import WorkspaceInstaller
 from databricks.labs.ucx.installer import InstallationManager
+from databricks.labs.ucx.migration.table_migrate import TablesMigrate
```
Comment (nfx) [inline src/databricks/labs/ucx/cli.py:16]:
```suggestion
from databricks.labs.ucx.hive_metastore.migration import TablesMigrate
```

keep the `hive_metastore` package, don't introduce the `migration` package.

File: tests/unit/migration/test_table_migrate.py [hunk 2]
```diff
@@ -9,10 +9,10 @@
     MigrationCount,
     Table,
     TablesCrawler,
-    TablesMigrate,
 )
+from databricks.labs.ucx.migration.table_migrate import TablesMigrate
 
-from ..framework.mocks import MockBackend
+from tests.unit.framework.mocks import MockBackend
```
Comment (nfx) [inline tests/unit/migration/test_table_migrate.py:15]: revert.

File: src/databricks/labs/ucx/cli.py (hunk 1 above)
Comment (FastLee) [reply src/databricks/labs/ucx/cli.py:16]: OK, scrapping this idea.
It thought to keep it on par with "Assessment"

File: tests/unit/migration/test_table_migrate.py (hunk 2 above)
Comment (FastLee) [reply tests/unit/migration/test_table_migrate.py]: Fixed

**Pull request merged**